include/cuda/std/detail/libcxx/include/type_traits

```diff
@@ -4767,6 +4767,8 @@ struct __has_operator_addressof
 
 #endif  // _LIBCUDACXX_CXX03_LANG
 
+template <class...> using _Void_t = void;
```

Comment:
I believe this should not be necessary anymore because we are always at least in C++14.

Please remove all changes to this file and switch from `_Void_t` to `void_t`

Reply:
I believe this came from the `std::span` PR.

Reply:
You are absolutely right, I think it should already be gone from there